src/main/java/org/jabref/logic/formatter/bibtexfields/ShortenDOIFormatter.java

```diff
@@ -0,0 +1,51 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import org.jabref.logic.importer.util.ShortDOIService;
+import org.jabref.logic.importer.util.ShortDOIServiceException;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.cleanup.Formatter;
+import org.jabref.model.entry.identifier.DOI;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ShortenDOIFormatter extends Formatter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ShortenDOIFormatter.class);
+
+    @Override
+    public String getName() {
+        return Localization.lang("Shorten DOI");
+    }
+
+    @Override
+    public String getKey() {
+        return "short_doi";
+    }
+
+    @Override
+    public String format(String value) {
+        ShortDOIService shortDOIService = new ShortDOIService();
+
+        DOI doi = null;
+
+        try {
+            doi = new DOI(value);
```

Comment:
It is preferable to use `DOI.parse(value)` since it adds a bit of cleanup and you don't need to catch the `IlegalArgumentException`.
https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/model/entry/identifier/DOI.java#L97

Reply:
Good point. Applied.